test/e2e/performanceprofile/functests/utils/daemonset/daemonset.go

```diff
@@ -43,6 +47,29 @@ func IsRunning(ctx context.Context, cli client.Client, namespace, name string) (
 		}
 		return false, err
 	}
-	testlog.Infof("daemonset %q %q desired %d scheduled %d ready %d", namespace, name, ds.Status.DesiredNumberScheduled, ds.Status.CurrentNumberScheduled, ds.Status.NumberReady)
-	return ds.Status.DesiredNumberScheduled > 0 && ds.Status.DesiredNumberScheduled == ds.Status.NumberReady, nil
+	isRunning := ds.Status.DesiredNumberScheduled > 0 && ds.Status.DesiredNumberScheduled == ds.Status.NumberReady
+	if !isRunning {
```

Comment:
style: everything else being equal, please align the happy path to the left: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

Reply:
The happy path is `isRunning == true`

Reply:
I moved the logic into a separate function to make it more readable. I hope it's ok now